docs/mkdocs/docs/index.md

```diff
@@ -218,8 +220,8 @@ The `'test_frame'` DataFrame will be used for the remainder of this guide.
     * `DatetimeIndex`
     * `MultiIndex` composed of above supported types
     
-    Currently, ArcticDB only supports `append()`-ing to a `RangeIndex` with a continuing `RangeIndex` (i.e. the appending `RangeIndex.start` == `RangeIndex.stop` of the existing data and they have the same `RangeIndex.step`). If a DataFrame with a non-continuing `RangeIndex` is passed to `append()`, ArcticDB does _not_ convert it `Int64Index` like Pandas and will produce an error.
-    
+    Currently, ArcticDB allows `append()`-ing to a `RangeIndex` only with a continuing `RangeIndex` (i.e. the appending `RangeIndex.start` == `RangeIndex.stop` of the existing data and they have the same `RangeIndex.step`). If a DataFrame with a non-continuing `RangeIndex` is passed to `append()`, ArcticDB does _not_ convert it `Int64Index` like Pandas and will produce an error.
+
     Also note, the "row" concept in `head()/tail()` refers to the physical row, not the value in the `pandas.Index`.
```

Comment:
Just noticed the grammar in this line is off too. Shall we fix it while we're here?

```suggestion
    The "row" concept in `head()/tail()` refers to the row number, not the value in the `pandas.Index`.
```